<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="main_menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>449</y>
      <width>861</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="settings">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>238</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#081c16&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="new_game">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>238</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#081c16&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url('data/menu_ui/фон.png');</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="levels" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>861</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="third_level">
     <property name="geometry">
      <rect>
       <x>516</x>
       <y>20</y>
       <width>72</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#081c16&quot;;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="second_level">
     <property name="geometry">
      <rect>
       <x>396</x>
       <y>20</y>
       <width>72</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#081c16&quot;;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="first_level">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>20</y>
       <width>72</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>1</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: &quot;#081c16&quot;;
color:transparent</string>
     </property>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_to_menu">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>72</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>1</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: &quot;transparent&quot;;
color:transparent</string>
     </property>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>main_menu</zorder>
   <zorder>levels</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
